### PR TITLE
Documentation Sprint | Installing Modules

### PR DESCRIPTION
**Pull request by DonRichards**
**GitHub Issue**: N/A

# What does this Pull Request do?
Improves Documentation

# What's new?
Documentation + examples

# How should this be tested?
Just read.

# Additional Notes:
N/A

# Interested parties
@Islandora-CLAW/committers

### CODE REVIEW

File: docs/technical-documentation/install-enable-drupal-modules.md [hunk 1]
```diff
@@ -1,6 +1,19 @@
...
+```shell
+$ composer require "<vendor>/<package>:<version>"
 
+# Example
+$ composer require "islandora/islandora_defaults:1.0.0"
+```
+
+In the [Islandora 8 playbook](https://github.com/Islandora-Devops/claw-playbook), you can add a Drupal module's or theme's machine name to the `drupal_composer_dependencies` variable [here](https://github.com/Islandora-Devops/islandora-playbook/blob/master/inventory/vagrant/group_vars/webserver/drupal.yml). 
+
+To enable the Drupal module or theme, add the module machine name to the `drupal_enable_modules` variable as well.
```
Comment (dannylamb) [inline docs/technical-documentation/install-enable-drupal-modules.md:14]: I'd collapse this into the previous paragraph.  It stands out and that will make the image closer to its text and look more relevant.

**Review by dannylamb — CHANGES_REQUESTED** [review]
Looks good to me.  Just one small change requested.

**Comment by DonRichards** [comment]
I hope this resolves it.

File: docs/technical-documentation/install-enable-drupal-modules.md [hunk 2]
```diff
@@ -13,7 +13,6 @@ $ composer require "islandora/islandora_defaults:1.0.0"
 In the [Islandora 8 playbook](https://github.com/Islandora-Devops/claw-playbook), you can add a Drupal module's or theme's machine name to the `drupal_composer_dependencies` variable [here](https://github.com/Islandora-Devops/islandora-playbook/blob/master/inventory/vagrant/group_vars/webserver/drupal.yml).
```
Comment (dannylamb) [inline docs/technical-documentation/install-enable-drupal-modules.md]: Oops.  Sorry if that wasn't clear.  I meant this line, to collapse the paragraphs that talk about ansible variables.

Comment (DonRichards) [reply docs/technical-documentation/install-enable-drupal-modules.md]: Still not sure what you're looking for. Line #?

**Review by dannylamb — APPROVED** [review]

**Comment by dannylamb** [comment]
You got it.  Sry for dropping the ball on this one @DonRichards.

**Pull request merged**